### ISSUE TITLE
Name and default value of "dynamic_backup_path"

### ISSUE THREAD

**Issue by ajoaoff** [kytos/mef_eline#198]
This attribute of an EVC indicates if a dynamic path can or cannot be used, so maybe it should be called simple "dynamic_path".
Its default value is set to "False", but I think it should be "True" if there is no primary or secondary path defined.

**Comment by italovalcy** [comment]
Yes, I agree with the new name and expected behavior. If the request is for an intra-switch circuit (both UNIs are in the same switch), the path (dynamic or defined by the user) will not matter. If the request is for an inter-switch EVC, then either the path must be defined by the user or a dynamic path should be calculated (e.g., by pathfinder).

**Comment by italovalcy** [comment]
Another issue related to this is: when you create an EVC with primary_path field and dynamic_backup_path=True, mef_eline just use any dynamic path instead of setup the primary path as requested by the user.

**Comment by ajoaoff** [comment]
> Another issue related to this is: when you create an EVC with primary_path field and dynamic_backup_path=True, mef_eline just use any dynamic path instead of setup the primary path as requested by the user.

@italovalcy, I was unable to reproduce this behavior. Can you describe your environment and your queries?
